src/app/core/services/policy.service.ts

```
@@ -144,26 +146,34 @@ export class PolicyService {
   }
 
   getApprovalString(emails: string[]): string {
-    let approverEmailsRequiredMsg = 'Expense will need additional approval from ';
+    let approverEmailsRequiredMsg = this.translocoService.translate('services.policy.expenseNeedsAdditionalApproval');
     approverEmailsRequiredMsg += emails.map((email) => '<b>' + email + '</b>').join(', ');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)